.github/workflows/ort.yml

```diff
@@ -100,7 +96,7 @@ jobs:
                 ort:
                   analyzer:
                     allowDynamicVersions: true
-                    enabledPackageManagers: [Cargo, NPM, PIP]
+                    enabledPackageManagers: [Cargo, NPM, PIP, GradleInspector]
```

Comment:
They didn't wanted to work with another package managers in java? as extra
And a bit out of scope, but dont we want to add other package managers for python and node like conda and poetry for py and yarn for node?

Reply:
It's only for the configuration file that the ort tool will analyze. e.g. we already publish only to maven for java, but the project's build tool is gradle and thus the configuration files of the project that need to be analyzed are the bulild.gradle files, which are being covered with GradleInspector

Reply:
I see, so it for what we use and not what available. 
I guess for GO and Csharp we are waiting for first release? not better to start already to avoid dealing with stuff late? out of the scope, will approve anyway, but worth considering.

Reply:
if we can't test it - we can't add it :/ but I added it to the guide of how to add new wrapper to the ort workflow